kubernetes/templates/1-quarkus-monitor-kubernetes.yml

```diff
@@ -42,25 +40,24 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   annotations:
-    app.quarkus.io/build-timestamp: 2023-06-30 - 12:45:38 +0000
     prometheus.io/scrape: "true"
     prometheus.io/path: /metrics
     prometheus.io/port: "8080"
     prometheus.io/scheme: http
   labels:
     app.kubernetes.io/version: 1.0.0
     app.kubernetes.io/name: quarkus-monitor
+    app.openshift.io/runtime: quarkus
   name: quarkus-monitor
 spec:
-  replicas: 1
+  replicas: 2
```

Comment:
Default replica count is set to 2, so we can make EDA autoscale back to 2 if someone lowers it to 1.